### PR TITLE
MapRules Integration

### PR DESCRIPTION
**Pull request by abalosc1**

### CODE REVIEW

File: osmtm/static/js/project.js [hunk 1]
```diff
@@ -500,6 +504,14 @@ osmtm.project = (function() {
           u = u.replace('zoom', 'z');
           url += "&background=custom:" + encodeURIComponent(u);
         }
+        var attribution_config_id = $('#attribution_config_id').val();
+        // if a uuid is found, there are presets and validations to be had
+        if (/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i.test(attribution_config_id)) {
+          var presets = map_rules_url + '/' + attribution_config_id + '/presets/iD'
```
Comment (maxgrossman) [inline osmtm/static/js/project.js]: per [#4](https://github.com/radiant-maxar/maprules/issues/4) we'll want these urls to be separate from the ones used to add the modal (one for the ui, other for the api)

File: osmtm/static/js/project.js [hunk 2]
```diff
@@ -500,6 +504,14 @@ osmtm.project = (function() {
...
+          var validations = map_rules_url + '/' + attribution_config_id + '/rules/iD'
```
Comment (maxgrossman) [inline osmtm/static/js/project.js]: so clean! so readable! so not how I do gross string concat :see_no_evil:

**Review by maxgrossman — CHANGES_REQUESTED** [review]
:clap: :clap: :clap: 

This is super cool, awesome stuff.
Less changing the url for opening editors this is ready for :ship:

**Review by maxgrossman — APPROVED** [review]

**Comment by abalosc1** [comment]
Accidentally created this branch of master instead of develop

**Pull request closed without merging**